pkg/cmd/enclave_setup.go

```diff
@@ -158,6 +166,14 @@ func selectConfig(configs []models.ConfigInfo, selectedConfiguredProject bool, p
 		}
 	}
 
+	if len(configs) == 1 {
+		// the user is expecting to a prompt, so print a message instead
+		if promptUser {
+			utils.Log("Selected only available config")
```

Comment:
We should show the config name:

```
Selected only available config: dev
```

We could also bold the `Selected only available config:` part of the message.

Reply:
See my other comment